pom.xml

```diff
@@ -2372,7 +2399,7 @@
         <carbon.rule.version>4.4.9</carbon.rule.version>
         <carbon.rule.imp.pkg.version>[4.4.0, 4.5.0)</carbon.rule.imp.pkg.version>
         <!-- Synapse Version -->
-        <synapse.version>2.1.7-wso2v86</synapse.version>
+        <synapse.version>2.1.7-wso2v85-SNAPSHOT</synapse.version>
```

Comment:
shall we not downgrade the synapse version?

Reply:
We use 2.1.7-wso2v85-SNAPSHOT version of synapse for the jdk 11 upgrade work. Therefore, I changed this version.